.github/workflows/docker.yml

```diff
@@ -41,6 +41,24 @@ jobs:
           else
             echo git_branch="$GITHUB_REF_NAME" >> $GITHUB_OUTPUT
           fi
+      - name: Free Disk Space (Ubuntu)
+        # This step is required for some repositories 
+        # as 21GB limited space provided by github action
+        # gets exceeded during build process 
+        uses: jlumbroso/free-disk-space@main
```

Comment:
we can directly copy the file in this repo to not rely on 3rd party repository. What do you think?